docs/topics/auth/customizing.txt

```diff
@@ -840,6 +840,9 @@ The following forms are compatible with any subclass of
 * :class:`~django.contrib.auth.forms.SetPasswordForm`
 * :class:`~django.contrib.auth.forms.PasswordChangeForm`
 * :class:`~django.contrib.auth.forms.AdminPasswordChangeForm`
+* :class:`~django.contrib.auth.forms.UserCreationForm`
```

Comment:
I don't think this is correct. `UserCreationForm` has a `username` field which all subclasses of `AbstractBaseUser` may not have.

Reply:
You're right. I've been thinking about it these days and thought about creating the UserCreationForm based on the model's required fields.
If you thing it should be a good solution, I can handle it by the weekend.

Reply:
I'm not sure -- it might result in unexpected behavior changes. For example, the admin uses `django/contrib/auth/admin.py:    add_form = UserCreationForm`.